app/Http/Controllers/AdminController.php

```diff
@@ -12,6 +13,19 @@
 class AdminController extends Controller
 {
 
+    public function formularzRejestracji()
+    {
+        $lekarze = User::where('funkcja', User::LEKARZ)->get();
+
+        $is_admin = session()->get(User::ADMIN) === 'true';
+        if (!$is_admin) {
+            header("Location: " . URL::to('/'));
+            die();
+        }
+
+        return view('admin.rejestracja', compact('lekarze'));
+    }
```

Comment:
Przeniosłem kod z routa (a wcześniej z widoku) do kontrolera

Reply:
Co tam robi się?

Reply:
Nie mam pojęcia.

Przeniosłem ten kod 1:1 z routa, a wcześniej pisał go @hydrokortyzon, on powinien wiedzieć więcej

@hydrokortyzon

Reply:
Wymówki! Zamienienie die na return to nie refactor, to konieczność!

Reply:
Ok, racja. Wywalone
Redirect jeszcze można by zmienić na coś laravelowego, nieprawdaż?